Content.Shared/Body/Systems/SharedBodySystem.Body.cs

```diff
@@ -308,9 +308,9 @@ private void SetupOrgans(Entity<BodyPartComponent> ent, Dictionary<string, strin
         foreach (var part in parts)
         {
 
-            _gibbingSystem.TryGibEntityWithRef(bodyId, part.Id, GibType.Gib, GibContentsOption.Skip, ref gibs,
-                playAudio: false, launchGibs:true, launchDirection:splatDirection, launchImpulse: GibletLaunchImpulse * splatModifier,
-                launchImpulseVariance:GibletLaunchImpulseVariance, launchCone: splatCone);
+            _gibbingSystem.TryGibEntityWithRef(bodyId, part.Id, GibType.Drop, GibContentsOption.Skip, ref gibs,
```

Comment:
Once again, bodyparts not dropping when being gibbed is by design. If you want items to drop use the giblet system to spawn chunks of meat.